mlem/cli/main.py

```diff
@@ -484,3 +492,10 @@ def config_arg(
             obj = build_mlem_object(model, subtype, conf, file_conf)
 
     return obj
+
+
+def handle_traceback():
+    with stderr_echo():
+        traceback_output = traceback.format_exc()
+        echo(traceback_output)
+    raise typer.Exit(1)
```

Comment:
Not sure this is needed, python exception traceback already should go to stderr (`python -c "raise Exception()" > /dev/null` will print stderr). And we are loosing exception information (you needed to fix tests because of it)

Reply:
@mike0sv, should we just finish this? Looks like @itstargetconfirmed almost did it.